.github/workflows/test.yml

```diff
@@ -0,0 +1,253 @@
+name: Run distribution tests
+
+on:
+  pull_request:
+    branches:
+      - 8.x-[1-9]+.x
+    paths-ignore:
+      - '**.md'
+
+env:
+  DRUPAL_TESTING_COMPOSER_PROJECT: thunder/thunder-project
+  DRUPAL_TESTING_DATABASE_USER: root
+  DRUPAL_TESTING_DATABASE_PASSWORD: root
+  DRUPAL_TESTING_TEST_DUMP_FILE: dump.sql
+  DRUPAL_TESTING_DATABASE_ENGINE: mysql
+  DRUPAL_TESTING_TEST_CODING_STYLES: false
+  DRUPAL_TESTING_TEST_PROFILE: thunder
+  DRUPAL_TESTING_INSTALLATION_FORM_VALUES: "thunder_module_configure_form.install_modules_thunder_demo=NULL"
+  DRUPAL_TESTING_TEST_DEPRECATION: false
+  DRUPAL_TESTING_TEST_GROUP: Thunder
+  PUSHER_CREDENTIALS: ${{ secrets.PUSHER_CREDENTIALS }}
+  RIDDLE_TOKEN: ${{ secrets.RIDDLE_TOKEN }}
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          coverage: none
+          php-version: '7.3'
+          extensions: Imagick
+
+      - name: Start MySql service
+        run: |
+          sudo /etc/init.d/mysql start
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.composer/cache
+          key: ${{ runner.os }}-composer-cache-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-cache-
+
+      - name: Setup environment variables
+        run: |
+          echo "::add-path::$HOME/.composer/vendor/bin"
+
+      - name: Get build environment
+        run: composer global require thunder/drupal-testing:dev-feature/min-testing
+
+      - name: Prepare the build
+        run: test-drupal-project prepare_build
+
+      - name: Install suggested dependencies
+        run: composer require "thunder/thunder_performance_measurement:dev-master" "thunder/testsite_builder:dev-master" "drupal/media_entity_generic:^1.0" "drupal/riddle_marketplace:^3.0-beta2" "drupal/nexx_integration:^3.0" "valiton/harbourmaster:~8.1" --no-update --no-progress --working-dir=/tmp/test/thunder/install
+
+      - name: Build the docroot
+        run: test-drupal-project build
+
+      - name: Test for deprecations
+        run: test-drupal-project deprecation
+        continue-on-error: true
```

Comment:
Does this work? The next step will probably not start, as long as this fails

Reply:
```
jobs.<job_id>.steps.continue-on-error

Prevents a job from failing when a step fails. Set to true to allow a job to pass when this step fails.
```

Reply:
but I am talking about drupal-testing